go.mod

```diff
@@ -207,4 +207,4 @@ require (
 
 replace github.com/ethereum-optimism/optimism v1.7.2 => ./
 
-replace github.com/ethereum/go-ethereum v1.13.8 => github.com/kroma-network/go-ethereum v1.101308.3-0.20241209003634-d80c1f1bfb77
+replace github.com/ethereum/go-ethereum v1.13.8 => github.com/kroma-network/go-ethereum v1.101308.3-0.20241212085106-1e691c4eb53c
```

Comment:
While not required for this PR, it would be nice to set it to the latest commit in that branch: (30ca3a8f71e3e823f161257f860470028d152bb6)

Reply:
Yeah, we have some updates in geth in the last few days, but I did not bump it since the new changes are not required for node logics in this PR.